fortls/helper_functions.py

```diff
@@ -576,6 +576,8 @@ def get_var_stack(line: str) -> list[str]:
         final_var += line[section.start : section.end]
 
     if final_var is not None:
+        # refuse any spaces in class
```

Comment:
Comment is not accurate, the added line escapes white spaces around the access token.
I suggest the comment is removed.